pkg/less/less.go

```diff
@@ -45,7 +45,7 @@ const (
 
 type Less struct {
 	// src is the source file being displayed.
-	src lineio.LineReader
+	src *lineio.LineReader
```

Comment:
does this belong to another PR?

Reply:
No, it's fixing a new go vet warning in Go 1.12.